contract_isp_invoice/contract.py

```diff
@@ -504,6 +504,15 @@ def _create_invoice(self, cr, uid, ids, context=None):
                 account_invoice_line_obj.write(
                     cr, uid, line_ids, {'invoice_id': inv[0]},
                     context=context)
+                # Assign analytic lines to first invoice
```

Comment:
This would require two queries instead of one, and considering how slow invoicing is, I would rather not.